frame/bridge/eth/relay/Cargo.toml

```diff
@@ -15,6 +15,8 @@ serde = { version = "1.0.106", optional = true }
 # darwinia
 darwinia-support = { default-features = false, path = "../../../support" }
 eth-primitives = { default-features = false, path = "../../../../primitives/eth-primitives" }
+# github
+ethereum-types = { default-features = false, git = "https://github.com/darwinia-network/parity-common.git" }
```

Comment:
This may be a little bit confusing for me since we note the `https://github.com/darwinia-network/substrate.git` as `substrate`.  

Maybe it is better to use `parity-common` than `github`.

Reply:
I think that's too annoy.

I only mark the main repo.